### PR TITLE
CLOUDP-138384: [cli] Update cobra2snooty

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes
_Jira ticket:_ CLOUDP-138384

## Checklist
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

We now have full coverage of args and they are correctly marked as required or optional based on the command usage documentation

### CODE REVIEW

**Review by andreaangiolillo — DISMISSED** [review]
LGTM

File: docs/mongocli/command/mongocli-atlas-privateEndpoints-azure-interfaces-describe.txt [hunk 1]
```diff
@@ -19,10 +19,26 @@ Syntax
...
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - endpointServiceId
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-atlas-privateEndpoints-azure-interfaces-describe.txt:37]: Do we need to remove this as an option on line 53 of this file if we make endpointServiceId the argument? Same question for all of the privateEndpoint interface commands.

File: docs/mongocli/command/mongocli-cloud-manager-backups-restores-list.txt [hunk 2]
```diff
@@ -23,6 +23,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-backups-restores-list.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-cloud-manager-backups-snapshots-list.txt [hunk 3]
```diff
@@ -23,6 +23,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-backups-snapshots-list.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-cloud-manager-clusters-describe.txt [hunk 4]
```diff
@@ -26,6 +26,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - id|name
+     - string
+     - true
+     - Name or ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-clusters-describe.txt:43]:
```suggestion
     - Name or ID of the cluster.
```

File: docs/mongocli/command/mongocli-ops-manager-backups-checkpoints-list.txt [hunk 5]
```diff
@@ -23,6 +23,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-backups-checkpoints-list.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-ops-manager-backups-config-describe.txt [hunk 6]
```diff
@@ -19,10 +19,26 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-backups-config-describe.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-ops-manager-backups-config-update.txt [hunk 7]
```diff
@@ -19,10 +19,26 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-backups-config-update.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-ops-manager-backups-restores-list.txt [hunk 8]
```diff
@@ -23,6 +23,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-backups-restores-list.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-ops-manager-backups-snapshots-list.txt [hunk 9]
```diff
@@ -23,6 +23,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-backups-snapshots-list.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-ops-manager-clusters-describe.txt [hunk 10]
```diff
@@ -26,6 +26,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - id|name
+     - string
+     - true
+     - Name or ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-ops-manager-clusters-describe.txt:43]:
```suggestion
     - Name or ID of the cluster.
```

File: docs/mongocli/command/mongocli-cloud-manager-backups-checkpoints-list.txt [hunk 11]
```diff
@@ -23,6 +23,22 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-backups-checkpoints-list.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-cloud-manager-backups-config-describe.txt [hunk 12]
```diff
@@ -19,10 +19,26 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-backups-config-describe.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-cloud-manager-backups-config-update.txt [hunk 13]
```diff
@@ -19,10 +19,26 @@ Syntax
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - clusterId
+     - string
+     - true
+     - ID of the cLuster.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-backups-config-update.txt:40]:
```suggestion
     - ID of the cluster.
```

File: docs/mongocli/command/mongocli-cloud-manager-metrics-disks-describe.txt [hunk 14]
```diff
@@ -23,6 +23,26 @@ Syntax
...
+     - Required
+     - Description
+   * - hostId
+     - string
+     - true
+     - Process identifier.
+   * - name
+     - string
+     - true
+     - partition name.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-cloud-manager-metrics-disks-describe.txt:44]:
```suggestion
     - Partition name.
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
Some typo fixes and question about privateEndpoints redundant arg/option

File: docs/mongocli/command/mongocli-atlas-privateEndpoints-azure-interfaces-describe.txt (hunk 1 above)
Comment (gssbzn) [reply docs/mongocli/command/mongocli-atlas-privateEndpoints-azure-interfaces-describe.txt:37]: good question, I'm not sure, this set of command on private endpoint are weird and the way they were coded doesn't match how they were documented, I need to test them out to see how they really work

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Pull request merged**